examples/machine-learning/README.md

```diff
@@ -94,72 +82,103 @@ gcloud storage buckets create gs://${BUCKET} --project=${PROJECT_ID} \
 gcloud storage buckets update gs://${BUCKET} --versioning
 ```
 
+Modify all 3 blueprints to configure the new bucket to serve as a Terraform
+remote backend:
+
+```yaml
+terraform_backend_defaults:
+  type: gcs
+  configuration:
+    bucket: customer-bucket # modify to bucket created above
+```
+
+### Set default values
+
+Modify the the blueprint deployment variables `project_id`, `region`, `zone`, in
+the `vars` block of all 3 blueprints:
+
+```yaml
+  project_id: customer-project
+  region: customer-region
+  zone: customer-zone
+```
+
+### Set kernel-patched OS image
+
+Obtain values for `source_image_project_id` and `source_image` from your Google
+Cloud representative. Set them at approximately lines 33 and 34 of
+`ml-slurm-a3-1-image.yaml`. 
+
+```yaml
+  source_image_project_id: source-image-project-id # use value supplied by Google Cloud staff
+  source_image: source-image-name                  # use value supplied by Google Cloud staff
+```
+
+### Reservation created by Google
+
 If Google Cloud staff have created a reservation for you, then skip this step
-and use the reservation name visible in the Cloud Console.  Otherwise, create
-a compact placement policy attached to a reservation of A3 VMs.
+and proceed to [manual reservation creation](#manual-creation-of-reservation).
 
-First, create a compact placement policy:
+In this scenario, you must alter the blueprint deployment variables
+`a3_reservation_name` and `a3_maintenance_interval` beginnning at approximately
+line 38 of `ml-slurm-a3-2-cluster.yaml`. Use the reservation name provided by
+Google and set the maintenance interval parameter to `PERIODIC`.
 
-```shell
-gcloud beta compute resource-policies create group-placement ${POLICY} \
-    --region=${REGION} \
-    --collocation=COLLOCATED \
-    --project=${PROJECT_ID}
+```yaml
+  # a3_reservation_name should be empty string by default; if Google staff
+  # have provided you with a reservation, supply it here
+  a3_reservation_name: reservation-name-provided-by-google
+  # a3_maintenance_interval should be empty string by default; if Google staff
+  # have provided you with a reservation, set it to PERIODIC
+  a3_maintenance_interval: PERIODIC
 ```
 
-And then create the reservation with the policy attached:
+### Manual creation of reservation
+
```

Comment:
It may be easier to invert these options and put manual creation before reservation created by Google

Reply:
I chose to put it in this order to avoid the "reader is blindly copying-and-pasting" problem. The README renders in GitHub UI with a very easy copy button.

I'm happy to reconsider if you feel strongly, but the choice was deliberate.